dev-docs/bidders/optidigital.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: bidder
+title: Optidigital
+description: Prebid Optidigital Bidder Adapter
+biddercode: optidigital
+pbjs: true
+floors_supported: true
+tcf2_supported: true
+usp_supported: true
+schain_supported: true
+media_types: banner
+gvl_id: 915
+sidebarType: 1
```

Comment:
for publishers & sales houses the following meta data is very helpful to vet new bidders upfront

- `ortb_blocking_supported`: true/partial/false
- `safeframes_ok`: true/false

Reply:
These parameters were added in the last commit.